plugins/foreman_openscap/1.0/index.md

```diff
@@ -0,0 +1,574 @@
+---
+layout: plugin
+title: Foreman OpenSCAP manual
+images: /plugins/foreman_openscap/1.0
+version: 1.0.0
+version_short: '1.0'
+foreman_version: '1.23'
+foreman_ansible_version: '3.x'
+# uncomment for older versions than stable or nightly
+robots: noindex
```

Comment:
This should be commented here, i meant uncomment it on the older versions of the manual so google stops indexing them and indexes this version instead